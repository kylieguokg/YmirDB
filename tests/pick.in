pick a -1
set a 1 2 3 4 5 6
get a
pick a -1
len a
pick a 10
pick a 2
get a
pop a
pick a 0
pick a 6
pick a 3
set b 29
append a b
list entries
pick a 6
pick b 1
bye
