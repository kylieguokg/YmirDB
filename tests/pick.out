> no such key

> ok

> [1 2 3 4 5 6]

> index out of range

> 6

> index out of range

> 2

> [1 2 3 4 5 6]

> 1

> index out of range

> index out of range

> 4

> ok

> ok

> b [29]
a [2 3 4 5 6 b]

> b

> 29

> bye
